chain/jsonrpc/src/lib.rs

```diff
@@ -18,7 +18,7 @@ use message::Message;
 use near_client::{ClientActor, GetBlock, Query, Status, TxDetails, TxStatus, ViewClientActor};
 use near_network::{NetworkClientMessages, NetworkClientResponses};
 use near_primitives::hash::CryptoHash;
-use near_primitives::serialize::{BaseEncode, from_base};
+use near_primitives::serialize::{BaseEncode, from_base, from_base64};
```

Comment:
why do we have both `from_base` and `from_base64`?

Reply:
base64 for transactions, base58 for other requests